actions/push-url.js

```diff
@@ -2,36 +2,7 @@ var program = require('commander');
 var request = require('request');
 
 var credentialLib = require('../libraries/credential');
-var requestHandler = require('../handlers/request');
-
-function createPushData (message, url, credential) {
-	var requestPushData = {
-		method: 'POST',
-		url: 'https://go.urbanairship.com/api/push/',
-		
-		json: {
-			'audience': 'all',
-			'notification': {
-				'alert': message,
-				'actions': {
-					'open': {
-						'type': 'url',
-						'content': url,
-					},
-				},
-			},
-			'device_types': 'all',
-		},
-
-		headers: {
-			'Authorization': 'Basic ' + credentialLib.encode(credential),
-			'Accept': 'application/vnd.urbanairship+json; version=3;',
-			'Content-Type': 'application/json',
-		},
-	}
-
-	return requestPushData;
-}
+var notification = require('../services/notification');
```

Comment:
You could use `var notificationService` instead  of `var notification` because It's more legible.